src/workloads/docs/LoggingActor.yml

```diff
@@ -0,0 +1,12 @@
+SchemaVersion: 2018-07-01
```

Comment:
`LoggingActorExample.yml`? So people know it's not meant to be run as a standalone workload?